specifications/transactions/onchain/README.md

```diff
@@ -1,10 +1,10 @@
 # On-Chain Data and Transactions
 
-Diem transactions mutate and create state (or resources) within the set of [on-chain modules](https://github.com/diem/diem/tree/master/language/diem-framework/modules), primarily the [Diem Account](https://github.com/diem/diem/blob/master/language/diem-framework/modules/doc/DiemAccount.md). The transaction format is defined in the [Move Adapter Specification](https://github.com/diem/diem/blob/master/specifications/move_adapter/README.md). Most participants of the Diem Payment Network (DPN) will submit SignedTransactions containing a script. Diem stores the scripts in [compiled form](https://github.com/diem/diem/tree/master/language/diem-framework/compiled/legacy/transaction_scripts) and in their [original form](https://github.com/diem/diem/tree/master/language/diem-framework/transaction_scripts). The DPN MainNet only allows this set of pre-registerd scripts to be submitted. Due to the evolving nature of Move and the Move compiler, compiling existing scripts may not result in the form stored in the directory stored above. Hence, it is recommended to use the compiled scripts.
+Diem transactions mutate and create state (or resources) within the set of [on-chain modules](https://github.com/diem/diem/tree/main/language/diem-framework/modules), primarily the [Diem Account](https://github.com/diem/diem/blob/main/language/diem-framework/modules/doc/DiemAccount.md). The transaction format is defined in the [Move Adapter Specification](https://github.com/diem/diem/blob/main/specifications/move_adapter/README.md). Most participants of the Diem Payment Network (DPN) will submit SignedTransactions containing a script. Diem stores the scripts in [compiled form](https://github.com/diem/diem/tree/main/language/diem-framework/compiled/legacy/transaction_scripts) and in their [original form](https://github.com/diem/diem/tree/main/language/diem-framework/transaction_scripts). The DPN MainNet only allows this set of pre-registerd scripts to be submitted. Due to the evolving nature of Move and the Move compiler, compiling existing scripts may not result in the form stored in the directory stored above. Hence, it is recommended to use the compiled scripts.
```

Comment:
`https://github.com/diem/diem/tree/main/language/diem-framework/transaction_scripts` url says page not found